compiler/noirc_evaluator/src/acir/mod.rs

```diff
@@ -2071,6 +2071,7 @@ impl<'a> Context<'a> {
                 if matches!(
                     &dfg[*instruction],
                     Instruction::Binary(Binary { operator: BinaryOp::Sub, .. })
+                        | Instruction::Binary(Binary { operator: BinaryOp::UncheckedSub, .. })
```

Comment:
```suggestion
                    Instruction::Binary(Binary { operator: BinaryOp::Sub | BinaryOp::UncheckedSub, .. })
```

Should be able to do this to keep the patterns small btw.